packages/components/src/components/Table/Table.stories.jsx

```diff
@@ -12,13 +12,13 @@ limitations under the License.
 */
 
 import { action } from '@storybook/addon-actions';
-import { Dropdown } from 'carbon-components-react';
+import { Dropdown } from '@carbon/react';
 import {
-  Add16 as Add,
-  TrashCan32 as Delete,
-  Restart16 as Rerun,
-  Renew16 as RerunAll
-} from '@carbon/icons-react';
+  Add,
+  TrashCan as Delete,
```

Comment:
16 size for this icon now?

Reply:
yeah this is right, not sure why it was 32 before since the styles constrain it to 16px anyway